docs/intro.rst

```diff
@@ -40,15 +40,38 @@ respectively.
 
 Startup
 ^^^^^^^
+Then you can build all the services::
 
-Then you can start all the services::
+   $ docker-compose -f docker/docker-compose.dev.yml build
 
-   $ docker-compose -f docker/docker-compose.dev.yml up -d --build
+You need MusicBrainz database containing all the MusicBrainz music metadata for setting up your application. Data dumps provided from https://musicbrainz.org can be downloaded from https://musicbrainz.org/doc/MusicBrainz_Database/Download. ``mbdump.tar.bz2`` is the core MusicBrainz database including the tables for artist, release groups etc. ``mbdump-derived.tar.bz2`` contains annotations, user tags and search indexes. The core and the derived database covers almost all the data required for a CritiqueBrainz server.
```

Comment:
"almost all"? What does this mean? are there things that we won't be able to do with just these two files? What other files are needed to be able to do all development?